tls/tls.go

```diff
@@ -87,11 +89,20 @@ func (r *RootCAs) Type() string {
 }
 
 // SortTLSConfigurationPerEntryPoints converts TLS configuration sorted by Certificates into TLS configuration sorted by EntryPoints
-func SortTLSConfigurationPerEntryPoints(configurations []*Configuration, epConfiguration map[string]*DomainsCertificates) error {
+func SortTLSConfigurationPerEntryPoints(configurations []*Configuration, epConfiguration map[string]*DomainsCertificates, defaultEntryPoints []string) error {
 	if epConfiguration == nil {
 		epConfiguration = make(map[string]*DomainsCertificates)
 	}
 	for _, conf := range configurations {
+		if conf.EntryPoints == nil || len(conf.EntryPoints) == 0 {
+			certName := conf.Certificate.CertFile.String()
+			// Keep only 50 characters to generate a readable log
+			if len(certName) > 50 {
+				certName = certName[0:50]
+			}
+			log.Debugf("No entryPoint is defined to add the certificate %s, it will be added to the default entryPoints: %s", certName, strings.Join(defaultEntryPoints, ","))
```

Comment:
```go
if log.GetLevel() >= logrus.DebugLevel {
	certName := conf.Certificate.CertFile.String()
	// Keep only 50 characters to generate a readable log
	if len(certName) > 50 {
		certName = certName[:50]
	}
	log.Debugf("No entryPoint is defined to add the certificate %s, it will be added to the default entryPoints: %s", certName, strings.Join(defaultEntryPoints, ","))
}
```